packages/paste-core/components/input/src/NumberStep.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
```

Comment:
thought: this file name is kind of confuding because there's no NumberStep component in here. We can split this file into two files to follow our conventions, since Capitalized file names signify React components:
- IncrementButton.tsx
- DecrementButton.tsx

Reply:
I dig it